content/blog/any-terraform-provider/index.md

```diff
@@ -0,0 +1,171 @@
+
+We’ve embraced this approach in many other areas as well:
+
+* Pulumi enables deploying and managing the lifecycle of any [Kubernetes YAML](https://kubernetes.io/docs/concepts/overview/working-with-objects/) file or [Helm Chart](https://helm.sh/). There’s a great ecosystem and library of Helm charts, and we make those all available to Pulumi users without any wrappers or indirections.
+* We introduced the [Pulumi AWS CDK Adapter](https://www.pulumi.com/blog/aws-cdk-on-pulumi/) to enable Pulumi users to incorporate AWS CDK constructs directly into their Pulumi programs without rewriting in Pulumi first.
+* And, of course, Pulumi offers features for deploying and managing [Azure Resource Manager templates](https://www.pulumi.com/registry/packages/azure-native/api-docs/resources/deploymentatscope/) and [CloudFormation templates](https://www.pulumi.com/registry/packages/aws/api-docs/cloudformation/stack/), enabling easy deployment of existing cloud provider IaC artifacts.
+
+With the new support for Any Terraform Provider, we’re bringing this same experience to the Terraform and OpenTofu provider ecosystems.
+
+## How Does This Work?
+
+Two new technologies in the Pulumi open source project underpin support for using any Terraform or OpenTofu provider.
+
+**Parameterized Providers** offer support for building a single provider that can implement many different “flavors.” For example, the `terraform-provider` provider can be parameterized to provide support for any Terraform or OpenTofu provider.
+
+**Local Packages** offer the ability to generate a Pulumi SDK for the language of your Pulumi project into a local folder, which can be directly incorporated into your project via your existing package manager along with `pulumi install`.
+
+Local packages are versioned, and parameterized providers support many different formats for their parameters.  For example, the most general form of the command to use an existing Terraform provider is:
+
+```shell 
+$ pulumi package add terraform-provider [<registry>/]<author>/<name> [version]
+```
+
+Or:
+
+```shell
+$ pulumi package add terraform-provider /path/to/my/terraform-provider-binary
+```
+
+## What’s Next?
+
+Today’s release is a big step in opening up access to even more providers for Pulumi users.  But there is still a lot more we have planned to build on top of these new foundations.  
+
+First, with locally generated SDKs, the documentation is available within the language SDK you generate (and so lights up in your IDE or development environment!). Still, there is no static docs page available in the Pulumi Registry to find and link to on the web.  In the future, we plan to offer the ability to publish documentation for locally bridged providers in the Registry.  In that case, the registry entry will be only the docs - the SDKs and provider binary will not need to be published/hosted in the Registry itself.  
```

Comment:
Should we link to https://github.com/pulumi/registry/issues/5302 here. That would allow us to gauge interest by volume of 👍.